lib/Dialect/SPIRV/Serialization/Serializer.cpp

```diff
@@ -1374,6 +1476,9 @@ LogicalResult Serializer::processOperation(Operation *op) {
   if (auto constOp = dyn_cast<spirv::ConstantOp>(op)) {
     return processConstantOp(constOp);
   }
+  if (auto funcCallOp = dyn_cast<spirv::FunctionCallOp>(op)) {
```

Comment:
Similarly, if we do `processOp<spirv::FunctionCallOp>` we don't need to explicitly register here.

Reply:
+1